<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGame</class>
 <widget class="QMainWindow" name="MainGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Arkanoid ;)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="gameGraphics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>460</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::SmartViewportUpdate</enum>
    </property>
    <property name="optimizationFlags">
     <set>QGraphicsView::DontSavePainterState</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Gra">
    <property name="title">
     <string>&amp;Gra</string>
    </property>
    <addaction name="act_newgame"/>
    <addaction name="separator"/>
    <addaction name="act_screen"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <addaction name="menu_Gra"/>
  </widget>
  <action name="act_newgame">
   <property name="text">
    <string>&amp;Rozpocznij grę..</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>&amp;Wyjście</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="act_settings">
   <property name="text">
    <string>&amp;Ustawienia</string>
   </property>
  </action>
  <action name="act_screen">
   <property name="text">
    <string>&amp;Screenshot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
